airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1123,16 +1123,21 @@ paths:
 
   /health:
     get:
-      summary: Checks if the API works
+      summary: Returns the status of Airflow's metadatabase and scheduler.
       operationId: airflow.api_connexion.endpoints.health_endpoint.get_health
       tags: [Monitoring]
       responses:
         '200':
-          description: It should always return "OK"
+          description: Returns info about metadatabase and last heart beat.
           content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HealthInfo'
             text/plain:
               schema:
                 type: string
+        '401':
+          $ref: '#/components/responses/Unauthenticated'
```

Comment:
This should be publicly available so that other services can easily use it
```suggestion
```

Reply:
Here is the change `5a215c8 `